sw/airborne/modules/stereocam/stereocam.c

```diff
@@ -76,6 +111,9 @@ extern void stereocam_disparity_to_meters(uint8_t *disparity, float *distancesMe
 
 extern void stereocam_start(void)
 {
+  struct FloatEulers euler = {RadOfDeg(STEREO_BODY_TO_STEREO_PHI), RadOfDeg(STEREO_BODY_TO_STEREO_THETA), RadOfDeg(STEREO_BODY_TO_STEREO_PSI)};
```

Comment:
For defines representing angles it is actually preferred if they are in radians. You can then easily specify them in degrees in your airframe file: `<define name="STEREO_BODY_TO_STEREO_PHI" value="90" unit="deg"/>`